.travis/utplsql_synonyms.sql

```diff
@@ -0,0 +1,48 @@
+whenever sqlerror exit failure rollback
+whenever oserror exit failure rollback
+set echo off
+set feedback on
+set heading off
+set verify off
+
+define ut3_owner   = "&1"
+define ut3_user    = "&2"
+define ut3_syntype = "&3"
+
+create &ut3_syntype synonym &ut3_user.ut_assert                       for &ut3_owner..ut_assert;
```

Comment:
Nice.
That was actually an ask form one of DBA's, to make sure that SYNONYMS do not use CREATE OR REPLACE so that we do not have a risk of overwriting synonyms for existing objects if there are conflicts of names.